devops/scripts/publish_bundle.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -e
+
+az acr login --name $TF_VAR_acr_name
+porter publish --registry "$TF_VAR_acr_name.azurecr.io" --debug
```

Comment:
--debug - same applies here
```suggestion
porter publish --registry "$TF_VAR_acr_name.azurecr.io"
```